package.json

```diff
@@ -97,45 +97,60 @@
         ]
     },
     "devDependencies": {
-        "@angular-devkit/build-angular": "15.2.10",
-        "@angular-devkit/core": "15.2.10",
-        "@angular-devkit/schematics": "15.2.10",
-        "@angular/animations": "15.2.10",
-        "@angular/cdk": "15.2.9",
-        "@angular/cli": "15.2.10",
-        "@angular/common": "15.2.10",
-        "@angular/compiler-cli": "15.2.10",
-        "@angular/core": "15.2.10",
-        "@angular/forms": "15.2.10",
-        "@angular/platform-browser": "15.2.10",
-        "@angular/router": "15.2.10",
+        "@angular-builders/custom-webpack": "16.0.1",
+        "@angular-devkit/build-angular": "16.2.12",
+        "@angular-devkit/core": "16.2.12",
+        "@angular-devkit/schematics": "16.2.12",
+        "@angular/animations": "16.2.12",
+        "@angular/cdk": "16.2.14",
+        "@angular/cli": "16.2.12",
+        "@angular/common": "16.2.12",
+        "@angular/compiler-cli": "16.2.12",
+        "@angular/core": "16.2.12",
+        "@angular/forms": "16.2.12",
+        "@angular/localize": "16.2.12",
+        "@angular/platform-browser": "16.2.12",
+        "@angular/platform-browser-dynamic": "16.2.12",
+        "@angular/platform-server": "16.2.12",
+        "@angular/router": "16.2.12",
+        "@ng-web-apis/universal": "3.0.7",
+        "@nguniversal/builders": "16.2.0",
+        "@nguniversal/express-engine": "16.2.0",
         "@nx/angular": "17.3.1",
         "@nx/jest": "17.3.1",
         "@nx/workspace": "17.3.1",
-        "@schematics/angular": "15.2.10",
+        "@schematics/angular": "16.2.12",
+        "@stackblitz/sdk": "1.9.0",
```

Comment:
This deps should be only inside demo's `package.json`:
https://github.com/taiga-family/taiga-ui/blob/8915881b55d50c0da9894fe9b0e3c43ee016ccd1/projects/demo/package.json#L17